pom.xml

```diff
@@ -29,6 +29,7 @@
         <project.github.repository>gluonhq/scenebuilder</project.github.repository>
         <repository.url>github.com:${project.github.repository}</repository.url>
         <jreleaser.files.active>RELEASE</jreleaser.files.active>
+        <jreleaser.previousTagName>17.0.0</jreleaser.previousTagName>
```

Comment:
Who updates the `previousTagName` when 18.0.0 is released?

Reply:
No. The value of previousTagName must be update explicitly. AFAICT you don't have yo specify it to generate a full changelog as JReleaser will use the last non early-access tag.

Reply:
That's what I was hoping JReleaser to do as well.

Reply:
Without `previousTagName`, running `mvn -pl :parent jreleaser:changelog` locally on the latest clone of this repo, results only in one line of changelog (which is the latest commit message).

```
## Changelog

- 2e59112 Removed null field and logic from Template enum (#516)
```

With `previousTagName`:

```
## Changelog

- 2e59112 Removed null field and logic from Template enum (#516)
- a05b8fd internal cleanup of Library (#517)
- a586380 Added copy code functionality to Color Picker (#496)
- 5e2e68a Added help menu option to show the welcome screen (#490)
- b9837ca Improved startup time (#447)

by delaying empty document creation and background loading tasks
- af569fc Updated license to 2022 (#515)
- 9faf4d7 SB test suite has fully migrated to JUnit 5 (#479)
- ec9ed97 Removed the white background from About and Register windows
- 0e13f47 Moved SceneBuilderTest into test package (#474)
- bb92623 Added clear community contribution guidelines (#504)
- 73a77be Add workflow for PR checks

Bringing back previously removed `build.yaml` with following chnages:
- 8b1aea9 Fix typo in jpackage job (#502)
- bcf9fc2 Fix early-access GH workflow (#500)
- d131458 Configure rolling releases (#458)
- 33d0424 Moves registration trigger to a help menu item. (#486)

* Relocates the registration functionality
- a8154ea Removed JCenter repository incl. related search. (#476)

Co-authored-by: Oliver Löffler <Oliver-Loeffler@users.noreply.github.com>
- 50de6a0 AbstractModalDialog shows correct application icon #459 (#477)

* Added title to save-or-not dialog.
- c9140f7 Updated the contents of the help menu to link into JavaFX community. (#401)

Co-authored-by: Oliver-Loeffler <Oliver-Loeffler@users.noreply.github.com>
- e2416fa Update Glisten dependencies (#471)

Co-authored-by: jose.pereda <jose.pereda@gluonhq.com>
- af0d9bc Rename FXML test resources following naming convention (#457)
- 06780ba Categories for Linux desktop entry (#460)
- 731dbf1 MacOS: avoid hiding SceneBuilder menu bar with FXML menu bar when useSystemMenuBar property is enabled (#405)

Co-authored-by: Oliver-Loeffler <Oliver-Loeffler@users.noreply.github.com>
- 7f45aed refactor public API to return Optional instead of null (#444)

* refactor public API to return Optional instead of null
- acccc10 All exception which occur during file open are now catched.
- 617b177 removed unused variable and method
- 119b37d added class javadoc and removed extra spaces
- af9f842 Removed spaces.
- e5c304b Added test case for FXOMLoader where an XMLStreamException is caused.
- 6b3e9d1 License header updated.
- a3439b2 refactor logTimestamp()
- adba6cf added a link to CONTRIBUTING.md
- a548ba3 FutureTask is now used instead of JavaFX task.
- 812c4bd JavaFX is now started per test.
- 656f1f5 Reworked test.
- 0d6789e FXML version is now used from FXMLLoader.
- 0f173a8 Removed error demo.
- b47bc49 Added another exception case, related to #327.
- 7d20993 initial draft of contribution workflow
- 974ee34 Error demonstration added.
- a46ca9f Added a test to show that errors caused by broken XML are detected as well.
- eb03ff4 Show Java version and runtime name in About Dialog (#415)
- 3cdd5a1 Fix URL for issue management (#414)
- 5b46313 Fix app name on Ubuntu (#407)
- 5dbfb50 Replaced the controller in the example with a generic name.
- f6d2a68 Fixed FXOMLoader so that no NPE is created in exception handling of load() method in case of unknown cause.
- 9082abd Added fix for #402 and prevents SceneBuilder from hiding errors during FXML loading.
```

It seems jreleaser starts collecting commits from its early-access pre-release when previousTagName is not used.

On a separate note, interestingly the commit messages are not formatted locally...

Reply:
Check your local git repository. Do you have an `early-access` tag? If so then that'd the reason for getting a shorter changelog while the project's version is still snapshot.

Reply:
Yes I have an `early-access` tag.

@abhinayagarwal I've removed `previousTagName`.